forc/Cargo.toml

```diff
@@ -29,6 +29,7 @@ forc-tracing = { version = "0.56.0", path = "../forc-tracing" }
 forc-util = { version = "0.56.0", path = "../forc-util" }
 fs_extra = "1.2"
 fuel-asm = { workspace = true }
+fuels-core = { workspace = true }
```

Comment:
This dependency here will likely create an issue for us. Due to our internal procedures we require forc not to depend on sdk. We should be able to move around this somehow. I will think about a different way of doing this but first I wanted to make sure this is not allowed by our CI.